x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/shared/SharedBlobCacheService.java

```diff
@@ -340,6 +345,12 @@ public SharedBlobCacheService(
         decayTask.rescheduleIfNecessary();
         this.rangeSize = SHARED_CACHE_RANGE_SIZE_SETTING.get(settings);
         this.recoveryRangeSize = SHARED_CACHE_RECOVERY_RANGE_SIZE_SETTING.get(settings);
+        this.evictedCountNonZeroFrequency = meter.registerLongCounter(
+            "blobCacheEvictionCountNonZeroFrequency",
+            "blobCacheEvictionCountNonZeroFrequency",
```

Comment:
It may be useful to have a description that is different from the name of the metric.